### PR TITLE
project root folder

### PR DESCRIPTION
**Pull request by ZimGil**
This pull request solves issue #3

### CODE REVIEW

**Comment by ortichon** [comment]
needs-work

File: src/app/serve/serve.component.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Component } from '@angular/core';
+import { timeout } from 'q';
```
Comment (ortichon) [inline src/app/serve/serve.component.ts]: ?

File: src/app/serve/serve.component.ts [hunk 2]
```diff
@@ -10,17 +11,21 @@ export class ServeComponent {
   private readonly MINIMUM_PORT = 1000;
   private readonly MAXIMUM_PORT = 99999;
 
+  rootFolder = "C:\\Development";
   hostname = "localhost";
   port = 4200;
   isOpen = true;
+  isForwardslash = false;
```
Comment (ortichon) [inline src/app/serve/serve.component.ts]: remove

File: src/app/serve/serve.component.ts [hunk 3]
```diff
@@ -10,17 +11,21 @@ export class ServeComponent {
...
 
   runServer(): void {
+    this.forwardSlashToBackslash();
```
Comment (ortichon) [inline src/app/serve/serve.component.ts]: remove

**Comment by ZimGil** [comment]
All unnecessary code was removed.
@ortichon Please review again.

File: src/app/serve/serve.component.ts [hunk 4]
```diff
@@ -18,9 +19,10 @@ export class ServeComponent {
     const request = {
       hostname: this.hostname,
       port: this.port,
-      isOpen: this.isOpen
+      isOpen: this.isOpen,
```
Comment (ortichon) [inline src/app/serve/serve.component.ts]: unused property?

File: src/app/serve/serve.component.ts [hunk 5]
```diff
@@ -18,9 +19,10 @@ export class ServeComponent {
...
+      rootFolder: this.rootFolder
     }
-    console.log('request: ', request);  
+    console.log('request: ', request);
```
Comment (ortichon) [inline src/app/serve/serve.component.ts]: redundant log

**Pull request merged**